packages/react-dom/src/server/ReactDOMServerFormatConfig.js

```diff
@@ -48,6 +48,10 @@ import hasOwnProperty from 'shared/hasOwnProperty';
 import sanitizeURL from '../shared/sanitizeURL';
 import isArray from 'shared/isArray';
 
+// Used to distinguish these contexts from ones used in other renderers.
+// E.g. this can be used to distinguish legacy renderers from this modern one.
+export const isPrimaryRenderer = true;
```

Comment:
I plan on building renderToString and Fizz in separate renderers so this will be used to deal with this case:

```js
function Foo() {
  renderToString(<Bar />);
}
renderToWritable(<Foo />);
```